src/js/api/config.js

```diff
@@ -119,7 +119,7 @@ const Config = function(options, persisted) {
     _copyToLocalization(allOptions);
     _deserialize(allOptions);
 
-    const language = getLanguage();
+    const language = allOptions.forceLocalizationDefaults ? Defaults.language : getLanguage();
```

Comment:
What is the value of `Defaults.language`? Why don't we just set a language in the config, and base the test on `setupConfig.language`? I much rather we use existing config options and support the setting of `language` in the config, than add a new option that no one need like `forceLocalizationDefaults`.

Reply:
`Defaults.language` is 'en'. I have mixed feelings about providing a config option to override language because we want to support google translate, which overrides the language tag on the page. If the player is setup after google translate then we read the updated language tag.
 if the language is being overriden via the config by the developer, then we have to count on them to update it when the page has been translated, which i doubt they would do.